### DISCUSSION TITLE
Detection config (detect, motion , record) are différent!

### DISCUSSION THREAD

**Discussion by seblang** [blakeblackshear/frigate#13290] [Report a Bug]
### Describe the problem you are having

in the UI interface, when I open one camera the detection config (top right) is not refleting the config I find in integration ! (it was ok in previews version)
![frigate](https://github.com/user-attachments/assets/e0c0e97c-580b-4518-91c2-24b4cc5b1a63)

### Steps to reproduce

just change the config in integration and check in UI!

### Version

0.14

### Frigate config file

```yaml
not relevant
```

### Relevant log output

```shell
nothing
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wired

### Camera make and model

HIKVISION

**Comment by NickM-27** [comment]
this has been fixed for 0.14.1
